internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -576,7 +576,9 @@ func (cf CloudFormation) errOnFailedStack(stackName string) error {
 	}
 	status := aws.StringValue(stack.StackStatus)
 	if cloudformation.StackStatus(status).IsFailure() {
-		return fmt.Errorf("stack %s did not complete successfully and exited with status %s", stackName, status)
+		return fmt.Errorf("stack %s did not complete successfully and exited with status %s.\n"+
```

Comment:
i think this part is already printed (the status/stack name) - maybe i'm wrong? but would be redundant if true

Reply:
So what we print right now (without any changes) [this](https://quip-amazon.com/NGlEAla1qHCR/Design-Doc-Copilot-App-Runner-Failed-Service-Management#temp:C:aRf41dbb31c7e0d46c383b930982). I'm just adding a small line after this.